src/Game/zMusic.cpp

```diff
@@ -1,19 +1,111 @@
+#include <types.h>
 #include "zMusic.h"
+#include "../Core/x/xSnd.h"
+#include "../Core/x/xMath.h"
+#include "zScene.h"
 
-#include <types.h>
+extern zMusicTrackInfo sMusicTrack[2];
+extern zVolumeInfo volume;
+extern uint32 sMusicPaused;
+extern LevelMusic sMusicLastEnum[2];
+extern float32 unkFloat; //0f by default.
+extern float32 minDelay; //Value is defaulted at 0.001f.
 
 // func_800A6E9C
 #pragma GLOBAL_ASM("asm/Game/zMusic.s", "volume_reset__Fv")
 
-// func_800A6EE4
-#pragma GLOBAL_ASM("asm/Game/zMusic.s", "zMusicRefreshVolume__Fv")
+//Reset both music tracks to their default volume.
+void zMusicRefreshVolume()
+{
+  	for (int32 i = 0; i < 2; i++)
+  	{
+  		zMusicTrackInfo* track = &sMusicTrack[i];
+  		if (track->snd_id != 0)
+  		{
+  			xSndSetVol(track->snd_id, volume.cur); //Only reset volume if the sound ID is not blank.
+  		}
+  	}
+}
 
 // func_800A6F50
 #pragma GLOBAL_ASM("asm/Game/zMusic.s", "zMusicInit__Fv")
 
 // func_800A7314
 #pragma GLOBAL_ASM("asm/Game/zMusic.s", "getCurrLevelMusicEnum__Fv")
 
+//Correct, but won't work due to the switch case jump table (messes with offsets).
```

Comment:
Instead of commenting out the function, I think this is slightly better:
```cpp
#if 0
// put entire function here as if it weren't commented out.
// also add any comments related to the non-matching status above the function here, but within the #if scope.
#else
// func_800A7414
#pragma GLOBAL_ASM("asm/Game/zMusic.s", "zMusicDo__Fi")
#endif
```
That way, anyone who comes along can simply change the `0` to `1` and mess with it without dealing with comments. This is kind of nitpicky, probably not that big of a deal, and we haven't come to a group decision about how to include non-matching functions in the source code, but I prefer it.

(After reading other formatting comment): Note that clang-format won't format code inside an unreachable directive, so before you set it to `#if 0`, format the code inside it first.